src/basic/vx_cal/unix_to_mdyhms.cc

```diff
@@ -105,18 +105,17 @@ int month_name_to_m (const char *month_str)
 {
 
 int month = -1;
-int str_len = strlen(month_str);
-char t_month_str[str_len + 1];
 
-for (int idx=0; idx<str_len; idx++) {
-   t_month_str[idx] = (idx==0) ? toupper((char)month_str[idx])
-                               : tolower((char)month_str[idx]);
-}
-t_month_str[str_len] = 0;
+// Convert to camel case
+string t_month_str(month_str);
+transform(t_month_str.begin(), t_month_str.begin() + 1,
+          t_month_str.begin(), ::toupper);
+transform(t_month_str.begin(), t_month_str.end(),
+          t_month_str.begin(), ::tolower);
```

Comment:
Two transforms should be swapped. Make lowercase first and make the first character upper.